### DISCUSSION TITLE
Thank you

### DISCUSSION THREAD

**Discussion by AmyJeanes** [WilliamVenner/fastgmad#2] [Show and tell]
Just wanted to give a thank you for saving my bacon with this tool - we were using the built-in one and running out of memory inside GitHub Actions and this has not only fixed it but massively sped up our CI runs

We're using it on our [TARDIS](https://github.com/MattJeanes/TARDIS) addon [GitHub Actions](https://github.com/MattJeanes/TARDIS/blob/dev/.github/workflows/ci.yml) to deploy beta/alpha versions to workshop automatically 😄 

So, thank you! Amazing work ❤️

**Comment by WilliamVenner** [comment]
You are most welcome :D I'm glad it was easy to deploy
